src/components/hack-details.js

```diff
@@ -36,66 +36,31 @@ const useStyles = makeStyles(theme => ({
 export function HackDetails(props) {
   const classes = useStyles();
 
-  const [data] = useState();
-  const [title, setTitle] = useState("");
-  const [description, setDescription] = useState("");
-  const [team, setTeam] = useState();
-  const [goal, setGoal] = useState("");
+  const [data, setData] = useState(null);
```

Comment:
its a bit of a nitpick, but I'd avoid using `data` as the name, and rather use something like `hackDetails` and `setHackDetails`... its more descriptive, and if more state come along will be easier to distinguish.